multirun/multirun.go

```diff
@@ -101,11 +103,11 @@ func (m multirun) boundedExecution(ctx context.Context) error {
 	return nil
 }
 
-func (m multirun) spawnWorker(ctx context.Context, commands <-chan command) error {
+func (m multirun) spawnWorker(ctx context.Context, commands <-chan command, errs chan<- error) bool {
```

Comment:
Would be good it leave a comment, briefly explaining what the boolean is. e.g. `returns true if an error was encountered`.

Reply:
Done.